static/app/views/settings/organizationMembers/inviteBanner.tsx

```diff
@@ -0,0 +1,186 @@
+  onSendInvite: (email: string) => void;
+};
+
+export function InviteBanner({missingMembers, onSendInvite}: Props) {
+  // TODO(cathy): include snoozing, docs link
+
+  const cards = missingMembers?.users.slice(0, 5).map(member => (
+    <MemberCard key={member.userId} data-test-id={`member-card-${member.userId}`}>
+      <MemberCardContent>
+        <MemberCardContentRow>
+          <IconGithub size="sm" />
+          <StyledExternalLink href={`http://github.com/${member.userId}`}>
+            {tct('@[userId]', {userId: member.userId})}
+          </StyledExternalLink>
+        </MemberCardContentRow>
+        <MemberCardContentRow>
+          <IconCommit size="xs" />
+          {tct('[commitCount] Recent Commits', {commitCount: member.commitCount})}
+        </MemberCardContentRow>
+        <Subtitle>{member.email}</Subtitle>
+      </MemberCardContent>
+      <Button
+        size="sm"
+        onClick={() => onSendInvite(member.email)}
+        data-test-id="invite-missing-member"
+        icon={<IconMail />}
+      >
+        {t('Invite')}
+      </Button>
+    </MemberCard>
+  ));
+
+  cards?.push(
+    <MemberCard key="see-more" data-test-id="see-more-card">
+      <MemberCardContent>
+        <MemberCardContentRow>
+          <SeeMoreContainer>
+            {tct('See all [missingMembersCount] missing members', {
+              missingMembersCount: missingMembers?.users.length,
+            })}
+          </SeeMoreContainer>
+        </MemberCardContentRow>
+        <Subtitle>
+          {tct('Accounting for [totalCommits] recent commits', {
+            totalCommits: missingMembers?.users.reduce(
+              (acc, curr) => acc + curr.commitCount,
+              0
+            ),
+          })}
+        </Subtitle>
+      </MemberCardContent>
+      <Button
+        size="sm"
+        priority="primary"
+        // TODO(cathy): open up invite modal
+        data-test-id="view-all-missing-members"
+      >
+        {t('View All')}
+      </Button>
+    </MemberCard>
+  );
+
+  return (
+    <StyledCard data-test-id="invite-banner">
+      <CardTitleContainer>
+        <CardTitleContent>
+          <CardTitle>{t('Bring your full GitHub team on board in Sentry')}</CardTitle>
+          <Subtitle>
+            {tct('[missingMemberCount] missing members that are active in your GitHub', {
+              missingMemberCount: missingMembers?.users.length,
+            })}
+            <Tooltip title="Based on the last 30 days of commit data">
+              <IconInfo size="xs" />
+            </Tooltip>
+          </Subtitle>
+        </CardTitleContent>
+        <ButtonContainer>
+          <Button
+            priority="primary"
+            size="xs"
+            // TODO(cathy): open up invite modal
+            data-test-id="view-all-missing-members"
+          >
+            {t('View All')}
+          </Button>
+        </ButtonContainer>
+      </CardTitleContainer>
+      <MemberCardsContainer>{cards}</MemberCardsContainer>
```

Comment:
if cards is empty, do we need any of this at all?

Reply:
i can return null if there are no members